apps/total-typescript/package.json

```diff
@@ -3,48 +3,63 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "build": "NODE_ENV=production next build",
-    "postbuild": "NODE_ENV=production next-sitemap",
     "dev": "next dev -p 3016",
-    "lint": "next lint",
-    "start": "next start -p 3016"
-  },
-  "prettier": {
-    "bracketSpacing": false,
-    "jsxBracketSameLine": false,
-    "printWidth": 80,
-    "semi": false,
-    "singleQuote": true,
-    "tabWidth": 2,
-    "trailingComma": "all",
-    "useTabs": false
+    "build": "next build",
+    "postbuild": "NODE_ENV=production next-sitemap",
+    "start": "next start -p 3016",
+    "lint": "eslint src/**/*.{ts,tsx} --fix && tsc --noEmit"
   },
   "dependencies": {
+    "@mdx-js/loader": "^2.1.2",
+    "@mdx-js/mdx": "^2.1.2",
+    "@mdx-js/react": "^2.1.2",
+    "@next/mdx": "^12.2.3",
+    "@skillrecordings/analytics": "workspace:*",
     "@skillrecordings/convertkit": "workspace:*",
-    "@skillrecordings/scripts": "workspace:*",
-    "@tailwindcss/forms": "^0.5.2",
-    "framer-motion": "^6.4.1",
-    "next": "^12.2.3",
+    "@skillrecordings/quiz": "workspace:*",
+    "@skillrecordings/react": "workspace:*",
+    "classnames": "^2.3.1",
+    "feed": "^4.2.2",
+    "lodash": "^4.17.21",
+    "next": "^12.2.2",
+    "next-compose-plugins": "^2.2.1",
     "next-sitemap": "^3.0.5",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-hook-form": "^7.29.0",
     "zod": "^3.14.4"
   },
   "devDependencies": {
-    "@tailwindcss/typography": "^0.5.2",
-    "@types/cookie": "^0.4.1",
-    "@types/jest": "^27.5.2",
+    "@next/env": "^12.2.2",
+    "@skillrecordings/scripts": "workspace:*",
+    "@skillrecordings/tsconfig": "workspace:*",
+    "@skillrecordings/types": "workspace:*",
     "@types/lodash": "^4.14.179",
+    "@types/mdx": "^2.0.2",
+    "@types/mdx-js__react": "^1.5.5",
     "@types/node": "^17.0.0",
     "@types/nodemailer": "^6.4.4",
     "@types/react": "^18.0.15",
     "@types/react-dom": "^18.0.6",
     "autoprefixer": "^10.4.2",
     "eslint": "^8.19.0",
     "eslint-config-custom": "workspace:*",
+    "lint-staged": "^11.2.6",
     "postcss": "^8.4.14",
-    "tailwindcss": "^3.1.4",
-    "typescript": "^4.7.4"
+    "prettier": "^2.6.2",
+    "tailwindcss": "^3.1.6",
+    "typescript": "^4.7.4",
+    "webpack": ">=4.3.0 <5.0.0"
+  },
+  "lint-staged": {
+    "*.{js,json,md,mdx,ts,tsx}": [
+      "prettier --write"
+    ],
+    "src/**/*.{ts,tsx,md,mdx,js}": [
+      "prettier --write"
+    ]
+  },
```

Comment:
this is all handled from the top level I believe so doesn't need to be in this package

Reply:
I noticed this is the case for more apps

Reply:
ya I removed it from the template recently since it's handled at the root level

we can triple check it, doesn't really matter in practice unless it doesn't work at all with this removed. then we need to update template